### PR TITLE
fix: ts

### PR DESCRIPTION
**Pull request by crazyair**

### CODE REVIEW

File: src/interface.tsx [hunk 1]
```diff
@@ -41,7 +41,8 @@ export type EventDataNode<TreeDataType> = {
   dragOverGapBottom: boolean;
   pos: string;
   active: boolean;
-} & TreeDataType;
+} & TreeDataType &
+  DataNode;
```
Comment (zombieJ) [inline src/interface.tsx:45]: BasicDataNode?

**Comment by crazyair** [comment]
怎么样了

**Comment by afc163** [comment]
来个用例？https://github.com/react-component/tree/blob/master/tests/type.spec.tsx

**Comment by crazyair** [comment]
> 来个用例？https://github.com/react-component/tree/blob/master/tests/type.spec.tsx

之前有添加了啊？

**Pull request merged**